Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/detail/AgendaDetailFragment.kt

```diff
@@ -246,6 +247,21 @@ class AgendaDetailFragment : Fragment() {
                 scheduleDetail.listRow.talkDescription.getHtmlFormattedSpanned()
 
         tv_agenda_detail_description.movementMethod = LinkMovementMethod.getInstance()
+        tv_agenda_detail_shareText.text = resources.getString(R.string.sharing_twitter_title)
+        imgv_twitter.setOnClickListener {
+            val twitter = viewModel.getSpeaker(scheduleDetail.listRow.primarySpeakerName)?.socialProfiles?.get("twitter")
+            var twitterVal = ""
```

Comment:
we could combine this `var` declaration with it's assignment
```suggestion
            var twitterVal = if(twitter.isNullOrEmpty()) scheduleDetail.listRow.primarySpeakerName else "@$twitter"
```

Reply:
We can also change it to a `val` in that case, right?